docs/diagrams/DuplicateTransactionSequenceDiagram.puml

```diff
@@ -9,7 +9,7 @@ participant ":DuplicateTransactionCommandParser" as DuplicateTransactionCommandP
 participant ":TransactionDescriptor" as TransactionDescriptor LOGIC_COLOR
 participant "dt:DuplicateTransactionCommand" as DuplicateTransactionCommand LOGIC_COLOR
 participant ":CommandResult" as CommandResult LOGIC_COLOR
-participant ":EditTransactionCommand" as EditTransactionCommand LOGIC_COLOR
+participant "<<class>> EditTransactionCommand" as EditTransactionCommandClass LOGIC_COLOR
```

Comment:
Could we do "<<class>>\nEditTransactionCommand" to follow the format from the textbook?